string order_id
string station_id
string drinks
string name
move_base_msgs/MoveBaseActionGoal goal
